integration-testcases/src/test/scala/org/apache/spark/testsuite/allqueries/AllDataTypesTestCase1.scala

```diff
@@ -129,13 +129,66 @@ class AllDataTypesTestCase1 extends QueryTest with BeforeAndAfterAll {
     catch {
       case e: Exception => print("ERROR : " + e.getMessage)
     }
+    try {
```

Comment:
Can you add the testcase under incubator-carbondata/integration/spark instead of here.